Categories/Morphism/IsoEquiv.agda

```diff
@@ -68,3 +69,59 @@ Isos = record
       }
   }
   where open Equiv
+
+----------------------------------------------------------------------
+
+-- An alternative notion of equality on isomorphisms that only
```

Comment:
Could you please split this into its own PR?

Reply:
Done in #51.